packages/fast-components-styles-msft/src/design-system/index.ts

```diff
@@ -13,6 +13,16 @@ export interface DesignSystem {
      */
     backgroundColor: string;
 
+    /**
+     * The neutral color, which the neutralPalette is based on

```

Comment:
Can you add a note to both of these that changes to the palette should also update these values? I want to make it super clear in documentation that changes to base colors and palettes should be synchronized.